packages/fxa-profile-server/lib/routes/avatar/post-from-auth-server.js

```diff
@@ -0,0 +1,65 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const assert = require('assert');
+
+const Joi = require('@hapi/joi');
+
+const config = require('../../config');
+const db = require('../../db');
+const hex = require('buf').to.hex;
+const img = require('../../img');
+const notifyProfileUpdated = require('../../updates-queue');
+const workers = require('../../img-workers');
+const avatarShared = require('./_shared');
+
+const FXA_PROVIDER = 'fxa';
+const FXA_URL_TEMPLATE = config.get('img.url');
+assert(
+  FXA_URL_TEMPLATE.indexOf('{id}') !== -1,
```

Comment:
I think you can also use [`String#includes()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes) here; `FXA_URL_TEMPLATE.includes('{id}'),`.